.github/workflows/phpunit.yml

```diff
@@ -41,6 +41,16 @@ jobs:
           path: plugin-shopware-6
 
       - name: Setup Node 14
+        if: |
+          matrix.shopware-versions == 'v6.4.0.0' ||
+          matrix.shopware-versions == 'v6.4.1.0' ||
+          matrix.shopware-versions == 'v6.4.2.1' ||
+          matrix.shopware-versions == 'v6.4.3.1' ||
+          matrix.shopware-versions == 'v6.4.4.0' ||
+          matrix.shopware-versions == 'v6.4.5.1' ||
+          matrix.shopware-versions == 'v6.4.6.0' ||
+          matrix.shopware-versions == 'v6.4.7.0' ||
+          matrix.shopware-versions == 'v6.4.8.2'
```

Comment:
Interesting, why was this not necessary for older versions e.g. `6.3.x.x`?

Reply:
We don't run JS tests for legacy Shopware versions, which includes everything below 6.4. As we don't run any JS tests, we also don't install any JavaScript dependencies, therefore this change wasn't required for `phpunit-legacy.yml`.